be2-scala/src/security/test/private_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
do you really want to commit these keys ?

Reply:
They are now removed by default and need to be generated before running the server (as documented in the `readme` now).